src/bindgen/config.rs

```diff
@@ -441,6 +444,12 @@ impl EnumConfig {
         }
         self.derive_tagged_enum_copy_constructor
     }
+    pub(crate) fn private_default_tagged_enum_constructor(&self, annotations: &AnnotationSet) -> bool {
+        if let Some(x) = annotations.bool("private-default-tagged-enum-constructor") {
+            return x;
+        }
```

Comment:
please add this to the list of enum annotations:

https://github.com/eqrion/cbindgen/blob/master/docs.md#enum-annotations

Reply:
Done.